client/src/app/pages/main/header/header.component.html

```diff
@@ -7,6 +7,17 @@ <h2 class="text-white mx-auto mt-4 mb-5">A free, standalone movement-focused pla
         <div>
           <iframe src="https://store.steampowered.com/widget/669270/" style="border: none;" frameborder="0" width="70%" height="190"></iframe>
         </div>
+        <a class="donate" href="https://opencollective.com/momentum-mod" target="_blank">
+          <div class="donateBody">
+            <span class="donateText">DONATE TO OUR COLLECTIVE</span>
```

Comment:
Let's change this wording to "SUPPORT US ON OPENCOLLECTIVE". Not sure how it'll affect the sizing or if this will properly size to fit.

Reply:
I had to widen the widget a bit to fit the text on one line. I also made it so if the screen width gets too small (<380px) it'll shorten the widget and put the text in 2 lines. Let me know if it looks fine like that.

Reply:
Yep it looks great!